microsoft-edge/webview2/releasenotes.md

```diff
@@ -24,7 +24,7 @@ The WebView2 team is delivering updates to the [WebView2 SDK][NuGetGallery] on a
 
 Release Date: Oct 19, 2020  
 
-[NuGet package][NuGetGallery1.0.674-prerelease] \| minimum Microsoft Edge version 88.0.674.0.
+[NuGet package][NuGetGallery1.0.674-prerelease] \| minimum WebView2 Runtime version 88.0.674.0.
```

Comment:
The min version is 86.0.616.0 for both this SDK and the other 1.0.622.22 SDK

Reply:
Is the min version of 86.0.616.0 true for both release and pre-release packages?